src/source/Rtcp/RtcpPacket.c

```diff
@@ -79,3 +79,66 @@ STATUS rtcpNackListGet(PBYTE pPayload, UINT32 payloadLen, PUINT32 pSenderSsrc, P
     LEAVES();
     return retStatus;
 }
+
+// Assert that Application Layer Feedback payload is REMB
+STATUS isRembPacket(PBYTE pPayload, UINT32 payloadLen)
+{
+    STATUS retStatus = STATUS_SUCCESS;
+    const BYTE rembUniqueIdentifier[] =  {0x52, 0x45, 0x4d, 0x42};
+
+    CHK(pPayload != NULL, STATUS_NULL_ARG);
+    CHK(payloadLen >= RTCP_PACKET_REMB_MIN_SIZE, STATUS_RTCP_INPUT_REMB_TOO_SMALL);
+    CHK(MEMCMP(rembUniqueIdentifier, pPayload + RTCP_PACKET_REMB_IDENTIFIER_OFFSET, SIZEOF(rembUniqueIdentifier)) == 0, STATUS_RTCP_INPUT_REMB_INVALID);
+
+CleanUp:
+
+    LEAVES();
+    return retStatus;
+}
+
+
+/**
+ * Get values from RTCP Payload
+ *
+ * Parameters:
+ *     pPayload         - REMB Payload
+ *     payloadLen       - Total length of payload
+ *     pMaximumBitRate  - REMB Value
+ *     pSsrcList        - buffer to write list of SSRCes into. If NULL return the expected size in pSize.
+ *     pSsrcListLen     - destination PUINT64 to store the size of the file when pBuffer is NULL;
+ */
+STATUS rembValueGet(PBYTE pPayload, UINT32 payloadLen, PDOUBLE pMaximumBitRate, PUINT32 pSsrcList, PUINT8 pSsrcListLen)
+{
+    ENTERS();
+    STATUS retStatus = STATUS_SUCCESS;
+    UINT8 ssrcListLen = 0, exponent = 0;
+    UINT32 mantissa = 0, i;
+    DOUBLE maximumBitRate = 0;
+
+    CHK(pPayload != NULL && pMaximumBitRate != NULL && pSsrcListLen != NULL, STATUS_NULL_ARG);
+    CHK(payloadLen >= RTCP_PACKET_REMB_MIN_SIZE, STATUS_RTCP_INPUT_REMB_TOO_SMALL);
+
+    memcpy(&mantissa, pPayload + RTCP_PACKET_REMB_IDENTIFIER_OFFSET + 4, sizeof(UINT32));
```

Comment:
MEMCPY and SIZEOF. Also + 4 - can this also be SIZEOF(UINT32)

Reply:
Fixed!